src/interactivity/base/InteractivityFactory.hpp

```diff
@@ -8,6 +8,8 @@
 #include "ApiDetector.hpp"
 
 #include "../inc/IInteractivityFactory.hpp"
+#include "PseudoConsoleWindowAccessibilityProvider.hpp"
+#include <wrl/implements.h>

```

Comment:
this may not be right; all the WRL stuff should be included already..? and `implements` is only necessary for `WRL::Implements<>` which lives in another file now!